### PR TITLE
For review hw 14

### PR DESCRIPTION
**Pull request by dezuz**

### CODE REVIEW

File: src/main/java/com/mateacademy/factory_method/BuilderCreator.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package com.mateacademy.factory_method;
```
Comment (Taniann) [inline src/main/java/com/mateacademy/factory_method/BuilderCreator.java]: [Package names are all lowercase, with consecutive words simply concatenated together (no underscores)](https://mate-academy.github.io/style-guides/java/java.html#s5.2.1-package-names)

File: src/main/java/com/mateacademy/factory_method/BuilderInterface.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package com.mateacademy.factory_method;
+
+public interface BuilderInterface {
```
Comment (Taniann) [inline src/main/java/com/mateacademy/factory_method/BuilderInterface.java]: do you really neeed this interface&

File: src/main/java/com/mateacademy/factory_method/BuilderProduct.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
+package com.mateacademy.factory_method;
+
+public class BuilderProduct {
```
Comment (Taniann) [inline src/main/java/com/mateacademy/factory_method/BuilderProduct.java]: add getters to BuilderProduct to have opportunity to get fields

**Review by Taniann — CHANGES_REQUESTED** [review]
good but please make a little fixing

File: src/main/java/com/mateacademy/factory_method/BuilderInterface.java (hunk 2 above)
Comment (dezuz) [reply src/main/java/com/mateacademy/factory_method/BuilderInterface.java]: my static nested class in BuilderProduct class implements this interface, so if we want to create one more class like BuilderProduct we just should implement this interface

**Review by Taniann — APPROVED** [review]
good

**Pull request merged**